aptos/scripts/publish_wormhole_message.ts

```diff
@@ -24,8 +24,8 @@ async function publishWormholeMessage(contractAddress: HexString, accountFrom: A
       TxnBuilderTypes.AccountAddress.fromHex(accountFrom.address()),
       BigInt(sequenceNumber),
       scriptFunctionPayload,
-      BigInt(1000), //max gas to be used
-      BigInt(1), //price per unit gas
+      BigInt(30000), //max gas to be used
+      BigInt(100), //price per unit gas
       BigInt(Math.floor(Date.now() / 1000) + 10),
```

Comment:
Don't we need to upgrade the SDK as well?

Reply:
I can successfully run this [publish_wormhole_message.ts](https://github.com/wormhole-foundation/wormhole/pull/1982/files/9fc796cfeeb7c84d1734349e6bc691a41cd6a7d1#diff-50635790a394f106497d7c2579155dc96abeb07e5330b38fae622fae7642fe42) without upgrade the SDK or other part of the code. But feel free to do so if needed. Thank you.